app/forms/cookies_form.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
```

Comment:
We usually add these when we have validations on the model which doesn't seem to be the case here. It would be less convoluted to just move this code to the controller action.